app/src/main/res/layout/fragment_movies_list.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/background_color">
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline_start"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_begin="20dp" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline_end"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_end="16dp" />
+
+    <TextView
+        android:id="@+id/movies_list_text_view"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="55dp"
+        android:drawableStart="@drawable/ic_combined_shape"
+        android:drawablePadding="8dp"
+        android:fontFamily="@font/roboto_black"
+        android:text="@string/movies_list"
+        android:textColor="@color/actor_name_color"
+        android:textSize="14sp"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toStartOf="@id/guideline_end"
+        app:layout_constraintStart_toEndOf="@id/guideline_start"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/movie_background_layout"
+        android:layout_width="170dp"
+        android:layout_height="296dp"
+        android:layout_marginTop="18dp"
+        android:background="@drawable/movie_bg_lout_preview_gradient"
+        app:layout_constraintStart_toEndOf="@id/guideline_start"
+        app:layout_constraintTop_toBottomOf="@id/movies_list_text_view">
+
+        <ImageView
+            android:id="@+id/movie_background_image_view"
+            android:layout_width="0dp"
+            android:layout_height="248dp"
+            android:layout_marginStart="2dp"
+            android:layout_marginTop="2dp"
+            android:layout_marginEnd="2dp"
+            android:onClick="onClickMoveToDetails"
+            android:src="@drawable/movie"
+            app:layout_constraintEnd_toEndOf="@id/movie_background_layout"
+            app:layout_constraintStart_toStartOf="@id/movie_background_layout"
+            app:layout_constraintTop_toTopOf="@id/movie_background_layout" />
+
+        <View
+            android:id="@+id/gradient_view"
+            android:layout_width="166dp"
```

Comment:
android:layout_width="166dp"
android:layout_height="248dp"

again, again, again:
relate to movie_background_image_view